bitrise.yml

```diff
@@ -488,9 +489,9 @@ workflows:
         - deploy_path: artifacts/
     meta:
       bitrise.io:
-        stack: osx-xcode-14.3.x-ventura
+        stack: osx-xcode-15.0.x
         machine_type_id: g2-m1.8core
-  L10nBuild:
+  Build:
```

Comment:
I would prefer not to modify this name since it will break the taskcluster integration.